Mathlib/Topology/Instances/ENNReal.lean

```diff
@@ -1361,6 +1361,32 @@ lemma liminf_const_sub (F : Filter ι) [NeBot F] (f : ι → ℝ≥0∞) {c : 
   (Antitone.map_limsSup_of_continuousAt (F := F.map f) (f := fun (x : ℝ≥0∞) ↦ c - x)
     (fun _ _ h ↦ tsub_le_tsub_left h c) (continuous_sub_left c_ne_top).continuousAt).symm
 
+lemma mul_liminf_le_liminf_mul {α : Type*} {f : Filter α} {u v : α → ℝ≥0∞} :
+    liminf u f * liminf v f ≤ liminf (u * v) f := by
+  refine mul_le_of_forall_lt fun a a_u b b_v ↦ (le_liminf_iff).2 fun c c_ab ↦ ?_
+  filter_upwards [eventually_lt_of_lt_liminf a_u, eventually_lt_of_lt_liminf b_v] with x a_x b_x
+  exact c_ab.trans (mul_lt_mul a_x b_x)
+
+lemma limsup_mul_le_mul_limsup {α : Type*} {f : Filter α} {u v : α → ℝ≥0∞}
```

Comment:
```suggestion
lemma limsup_mul_le {α : Type*} {f : Filter α} {u v : α → ℝ≥0∞}
```

Reply:
Done.

Reply:
Impossible, there's already a lemma called ENNREal.limsup_mul_le in Mathlib.Order.Filter.ENNReal. I've kept the initial name, and added a comment to explain that conflict.

Reply:
Given that the existing `ENNReal.limsup_mul_le` is the same statement under different assumptions, you should call this
```suggestion
lemma limsup_mul_le' {α : Type*} {f : Filter α} {u v : α → ℝ≥0∞}
```

Reply:
Done.